CRM/Utils/File.php

```diff
@@ -604,6 +604,9 @@ public static function absoluteDirectory($directory, $basePath = NULL) {
     // make everything absolute from the baseFilePath
     $basePath = ($basePath === NULL) ? self::baseFilePath() : $basePath;
 
+    // ensure that $basePath has a trailing slash
+    $basePath = (substr($basePath, -strlen(DIRECTORY_SEPARATOR)) != DIRECTORY_SEPARATOR) ? $basePath . DIRECTORY_SEPARATOR : $basePath;
```

Comment:
Yup tested in my local too, but still we need to wait until the test build passes, just in case to ensure there's no unintended regression.